forth/base.fs

```diff
@@ -80,7 +80,7 @@ parse-name asm included
    1 to foo
    foo . \ prints 1 )
 : value ( n -- )
-dup code lda,# 100/ ldy,#
+code split swap lda,# ldy,#
```

Comment:
How about `split ldy,# lda,#`?

Reply:
I tried that first, but I couldn’t get it to work, very tricky! Other words
assume that lda and ldy are in that order. At least TO does.

I agree it would be nicer to have it that way, to avoid the swap...

Reply:
ok, I think I fixed up all places that needed it :-)

Reply:
Nice! :+1:

Reply:
Not sure I like the change in retrospect... It does save three bytes, but for a little-endian platform, it feels ugly that the most significant byte would have the lower address.

Reply:
I tend to agree. I just saw the phrase `swap [use2] [use1]` and thought `[use1] [use2]` without a second thought for the consequences. More than the three bytes, is `value` used enough times in compiling the sources that 4 extra load/stores is even >0.1% time save? Probably not?

Reply:
I measured with VICE: the swap makes system compilation 3 jiffies slower. That is less than 0.1%.